javascript/javascript1/week1/Ez Namey.js

```diff
@@ -0,0 +1,8 @@
+// Startup name generator
+
+const randomNumber1 = Math.floor(Math.random() * 10);
```

Comment:
since renadomNumber1 and renadomNumber2 are doing the same thing why did you declare 2 const? you can use one const to get a random number.

Reply:
Another way, constants firstWords and secondWords will have same index